ddtrace/tracer/option.go

```diff
@@ -58,6 +61,12 @@ type config struct {
 	// all spans.
 	globalTags map[string]interface{}
 
+	// httpClientCodes specifies the range of HTTP client status codes considered as errors.
```

Comment:
These need to be in `globalconfig` because they need to be accessed by integrations. This config is for the tracer only.